cayman/functions.py

```diff
@@ -102,12 +102,11 @@ def run_proteome_annotation(args):
 
 	annotator = CazyAnnotator()
 	print("Reading HMMs")
-	annotator.read_hmms(os.path.join(args.hmmdb, "hmms"))  # path_to_hmm_file="/g/scb2/zeller/karcher/CAZY_project_v2/scripts/absolute_paths_all_cv_hmms_scratch.txt") # Good idea to move HMMs to scratch for faster input
+	annotator.read_hmms(os.path.join(args.hmmdb, "hmms"))
 	print("Reading sequences")
-	annotator.read_sequences(path_to_sequences=args.proteins)  # "/g/scb/zeller/karcher/kyanna_cazy_annotations/scripts/../results/prokka/086_scaffolds.faa")
+	annotator.read_sequences(path_to_sequences=args.proteins)
 	print("Annotating sequences (can take a few minutes; be patient)")
 	annotator.annotate_sequences_with_all_hmms(threads=args.threads)
 	print("Filtering and merging annotations over folds")
-	annotator.curate_annotations(precomputed_hmm_cutoffs=os.path.join(args.hmmdb, "cutoffs.csv"))  # "/g/scb2/zeller/karcher/CAZY_project_v2/results/CV_HMMs_optimal_cutoffs_all_v3.csv")
-	# breakpoint()
-	annotator.annotations_filtered.to_csv(args.output_file)  # "cazy_annotations.csv")
+	annotator.curate_annotations(precomputed_hmm_cutoffs=os.path.join(args.cutoffs))
```

Comment:
```suggestion
	annotator.curate_annotations(precomputed_hmm_cutoffs=args.cutoffs)
```

don't need the `join` any more